content/docs/ops/runbook/rotating-kubernetes.md

```diff
@@ -9,22 +9,43 @@ title: Rotating Secrets VI - Kubernetes
 
 ## Introduction
 
-Cloud.gov uses kubernetes to provided managed services, including elasticsearch and redis. The kubernetes bosh deployment and service broker are deployed from https://github.com/18F/cg-deploy-kubernetes.
+cloud.gov uses Kubernetes to provided managed services, including elasticsearch
+and redis. The Kubernetes bosh deployment and service broker are deployed from
+https://github.com/18F/cg-deploy-kubernetes.
+
+When rotating Kubernetes, you must open a scheduled maintenance window in
+Statuspage. Use the **Maint - GovCloud - Experimental services disruption**
+_Scheduled Maintenance_ template. Schedule the window to last at least three
+hours in order to complete all the steps below when rotating secrets in
+Kubernetes.
+
+Lastly, get familiar with the [Troubleshooting Kubernetes]({{< relref
+"docs/ops/runbook/troubleshooting-kubernetes.md" >}}) runbook in case
+anything goes wrong during the deployment.
 
 ## Rotating certificates
```

Comment:
This section is going to be a link to the main **Secrets I** docs around handling mutual TLS across all our deployments.